### DISCUSSION TITLE
Virus on a Network example

### DISCUSSION THREAD

**Discussion by malawleyGH** [mesa/mesa#2112] [Q&A]
Hi,
I am trying to run the 'virus on a network' example. I followed the instructions closely, and the server opens and displays the sliders and windows. I hit reset and then start hitting the step button. The simulation seems to be stepping but nothing is getting displayed, the windows remain blank.  Has anyone experienced this?

Here is some more information. When the browser opens, and I hit F12, I see this

Accessibility
Form elements must have labels: Element has no title attribute Element has no placeholder attribute
IDs used in ARIA and labels must be unique: Document has multiple elements referenced with ARIA with the same id attribute: fps
<html> element must have a lang attribute: The <html> element does not have a lang attribute
ARIA input fields must have an accessible name: Element has no title attribute
Compatibility
A 'viewport' meta element was not specified.

Also, 

runcontrol.js:171 
 WebSocket is already in CLOSING or CLOSED state.
runcontrol.js:171 
 WebSocket is already in CLOSING or CLOSED state.
runcontrol.js:171 
 WebSocket is already in CLOSING or CLOSED state.
runcontrol.js:171 
 WebSocket is already in CLOSING or CLOSED state.
runcontrol.js:171 
 WebSocket is already in CLOSING or CLOSED state.
[NEW] Explain Console errors by using Copilot in Edge: click 
 to explain an error. Learn more
Don't show again
runcontrol.js:171 
 WebSocket is already in CLOSING or CLOSED state.

I have no idea what this means, but perhaps someone does and can tell me if this is the problem.

**Comment by rht** [comment]
Have you tried different browsers that are less ARIA restrictive than Edge?
